library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -773,13 +799,17 @@ public Uri getBatchesUri() {
      * Get a parameterized SQL WHERE clause to select a bunch of IDs.
      */
     static String getWhereClauseForIds(long[] ids) {
```

Comment:
inline this method it's confusing that one use of `getWhere` calls the single param and one calls the double param, they should both just call the double param